src/components/ReportActionItem/ReportPreview.tsx

```diff
@@ -230,7 +230,7 @@ function ReportPreview({
             return '';
         }
 
-        if (ReportUtils.hasHeldExpenses(iouReport?.reportID) && canAllowSettlement) {
+        if (ReportUtils.hasHeldExpenses(iouReport?.reportID) && canAllowSettlement && !isNonHeldAmountNegative) {
```

Comment:
Here's an example of something not clear. Why don't we return `nonHeldAmount` here if `! isNonHeldAmountNegative`?

Reply:
@Beamanator As the original issue states, the `nonHeldAmount` amount can be negative if we hold a transaction. Since we cannot pay partially in this case, we shouldn't display the `nonHeldAmount` as the default option.

Reply:
That's a great comment for this PR 🙏  Can you please add a comment like that here? & think about the other places you made similar changes - the point is so that other people can read the code & understand why `isNonHeldAmountNegative` is needed so they don't have to find the PR that made the change, then find the issue, then read to figure out the context

Reply:
Added comments.